bugbug/tools/code_review.py

```diff
@@ -326,6 +328,10 @@ def get_all_inline_comments(
                 if transaction["authorPHID"] == "PHID-USER-cje4weq32o3xyuegalpj":
                     continue
 
+                # No comments
+                if not transaction["comments"]:
+                    continue
+
```

Comment:
If there are no comments present, we encounter an `IndexError` when accessing the `comments` array:
`transaction_comment = transaction["comments"][0]`

It would make sense to skip any transactions that do not contain any comments.

Reply:
But we already have `len(transaction["comments"]) != 1`, which should take care of this case.

Reply:
Removed in b68a1b33ce01854a6615c13bd321360b07adcc6b, as it has been resolved in #4266.